packages/migrate/src/index.ts

```diff
@@ -182,14 +192,14 @@ const Migrate = {
 
     if (Migrations.isDeployed() === false) return 0;
 
-    const migrationsOnChain = async migrationsAddress => {
+    const migrationsOnChain = async (migrationsAddress: string) => {
       return (
         (await Migrations.interfaceAdapter.getCode(migrationsAddress)) !== "0x"
       );
     };
 
     // Two possible Migrations.sol's (lintable/unlintable)
-    const lastCompletedMigration = migrationsInstance => {
+    const lastCompletedMigration = (migrationsInstance: any) => {
```

Comment:
Same as above

Reply:
Same response as the one related to typing `Migrations`.